app/views/avo/partials/_header.html.erb

```diff
@@ -1 +1,11 @@
-<%= link_to Avo.configuration.app_name, '/', class: 'text-primary-500 font-semibold', target: :_blank %>
+<div class="flex">
+  <%= link_to Avo.configuration.app_name, '/', class: 'text-primary-500 font-semibold', target: :_blank %>
+
+  <% if false %>
+    <div class="ml-12">
+      <% current_user.accounts.each do |account| %>
+        <%= link_to account.name, switch_account_path(account.id), class: class_names({"underline": session[:tenant_id].to_s == account.id.to_s}), data: {turbo_method: :put} %>
+      <% end %>
+    </div>
+  <% end %>
```

Comment:
@adrianthedev was this intentionally merged?

Reply:
This partial is the "ejectable" one. Not bad practice, wondering if we should keep it on this file so everyone have this piece of code when ejecting header or eject it on our dummy and modify it there for testing purposes.

Reply:
Pfff. You're right.

Can you please move that to the `dummy` partial?